changelog.md

```diff
@@ -29,6 +42,24 @@ Since this topic has so distinct requirements for different organizations, it is
 This version now implements all the necessary hooks needed to secure most external interfaces to your specific needs.
 If you are interested in a very simple implementation, you can look into the [Example Auth Plugin](https://github.com/mesosphere/marathon-example-plugins/tree/master/auth)
 
+#### Persistent Store Cache 
+All entities in the persistent store (ZK) are loaded into a cache during leader election.
+Subsequent reads are delivered from that cache. Updates to entities also update the cache.
+This cache should improve read access time significantly.
+Please be aware that the time until the leader takes over leadership will be increased with a lot of apps and tasks. 
```

Comment:
I don't understand what exactly this time period is, can we clarify it a bit? Is it the time between the old leader becoming unavailable and the new leader becoming available?